pkg/git/git.go

```diff
@@ -111,3 +111,45 @@ func Commit(logger *zap.SugaredLogger, revision, path string) (string, error) {
 	}
 	return strings.TrimSuffix(output, "\n"), nil
 }
+
+func SubmoduleFetch(logger *zap.SugaredLogger, path string) error {
+	// HACK: This is to get git+ssh to work since ssh doesn't respect the HOME
```

Comment:
This looks like this hack is copied from `Fetch()`.  Opportunity to de-dup by creating a helper func?

Reply:
@sbwsg indeed :stuck_out_tongue: I was a bit lazy 🤫, I need to enhance that :wink:

Reply:
nice :+1: